### PR TITLE
Add neofetch in Minecraft

### PR DESCRIPTION
**Pull request by zly2006**
### Description/描述

Neofetch is a command-line system information tool, with an ascii logo of your system distro.

This PR add this in Minecraft

### Related Issues/相关的 issue

n/a

### Checklist/清单

- [x] My changes generate no new warnings/我的修改没有产生新的警告
- [x] My code builds and runs on my machine/我的代码在我的电脑上可以编译和运行
- [ ] I documented my code/我已经为我的代码添加了注释/文档

### Screenshots

Screenshot

### CODE REVIEW

**Comment by Qubik65536** [comment]
Um... just call it "info fetch" or similar stuff, don't directly use `neofetch` if possible, it is sometimes great not to refer to another software.

**Comment by zly2006** [comment]
> Um... just call it "info fetch" or similar stuff, don't directly use `neofetch` if possible, it is sometimes great not to refer to another software.

how about `minefetch`?

**Comment by Qubik65536** [comment]
> > Um... just call it "info fetch" or similar stuff, don't directly use `neofetch` if possible, it is sometimes great not to refer to another software.
> 
> how about `minefetch`?

sure

**Comment by Qubik65536** [comment]
and do rename files in the code according to the new name

File: src/main/java/com/github/zly2006/reden/RedenClient.kt [hunk 1]
```diff
@@ -159,6 +159,19 @@ class RedenClient : ClientModInitializer {
...
+                            val width = renderer.getWidth(" ")
+                            (0x20..0x7E)
+                                .map { it.toChar() }
+                                .groupBy { renderer.getWidth(it.toString()) }
+                                .forEach {
+                                    println("${it.key}: ${it.value.joinToString("")}")
+                                }
+                            1
+                        }
+                    }
```
Comment (Qubik65536) [inline src/main/java/com/github/zly2006/reden/RedenClient.kt:174]: What is this command used for?

Comment (zly2006) [reply src/main/java/com/github/zly2006/reden/RedenClient.kt:174]: get chars that have the same width to generate ascii art

**Review by Qubik65536 — APPROVED** [review]

**Comment by zly2006** [comment]
Minecraft font is not in the same width, we can only get the width from the client, and the algorithm to compute width would be so complicated and problematic.

I think we should use network packets to compute on the client side, or use a monospaced font and add it to Minecraft using resourcepacks.

But there are still some problems:
1. mojang does not support rtf fonts well: https://forums.minecraftforge.net/topic/107600-possible-to-use-ttf-file-format-for-custom-font/
2. Try to find a pixelated monospaced font (i have no idea)
3. use mod

**Comment by zly2006** [comment]
totally broken(font: hack nerd mono)......

**Comment by zly2006** [comment]
> Minecraft font is not in the same width, we can only get the width from the client, and the algorithm to compute width would be so complicated and problematic.
> 
> I think we should use network packets to compute on the client side, or use a monospaced font and add it to Minecraft using resourcepacks.
> 
> But there are still some problems:
> 
> 1. mojang does not support rtf fonts well: https://forums.minecraftforge.net/topic/107600-possible-to-use-ttf-file-format-for-custom-font/
> 2. Try to find a pixelated monospaced font (i have no idea)
> 3. use mod

@lovexyn0827 @wafarm  两位有没有什么建议？关于等宽字符

**Pull request closed without merging**